website/data/docs-nav-data.json

```diff
@@ -382,8 +382,8 @@
         "path": "api-gateway"
       },
       {
-        "title": "Usage",
-        "path": "api-gateway/api-gateway-usage"
```

Comment:
Is there a way to alias this so we don't break any outstanding links?

Reply:
I believe this is an artifact of the base branch being rebased and a redirect was already added when this change was made.

Reply:
See https://github.com/hashicorp/consul/pull/12811 for how the redirect was added.